waddrmgr/address.go

```diff
@@ -263,6 +272,21 @@ func (a *managedAddress) ExportPrivKey() (*btcutil.WIF, error) {
 	return btcutil.NewWIF(pk, a.manager.chainParams, a.compressed)
 }
 
+// newAddress returns a new address based on the passed account and address
+// pubkeyhash.
+func newAddress(m *Manager, account uint32, addressID []byte) (*publicAddress, error) {
```

Comment:
If the `publicAddress` bits aren't separated out (see previous comment), this can simply be:

```Go
newManagedPublicAddress(m *Manager, account uint32, addressID []byte) (*managedAddress, error)
```